BilibiliLive/Component/Video/VideoDetailViewController.swift

```diff
@@ -320,8 +320,16 @@ class VideoDetailViewController: UIViewController {
         }
 
         if let season = data.View.ugc_season {
-            allUgcEpisodes = Array((season.sections.map { $0.episodes }.joined()))
+            if season.sections.count > 1 {
+                season.sections.first(where: { section in section.episodes.contains(where: { episode in episode.aid == data.View.aid }) }).map { section in
```

Comment:
do not use `map` for assigning value
```
if season.sections.count > 1,
  let episodeSession = season.sections.first(where:...) {
      allUgcEpisodes = episodeSession
} else {
    allUgcEpisodes = ...
}
```

Reply:
🤔 does the naming of season strictly need to be modified?